src/Artesian/GMEPublicOffers/_Enum/Market.py

```diff
@@ -23,3 +23,4 @@ class Market(Enum):
     MBh = 36
     MRR = 50
     MIXBID = 70
+    AFRR = 100
```

Comment:
@jacopocinaark please confirm server side integer value

Reply:
I just double checked in Arkive.Service and AFRR = 100 there.

Reply:
yes, I can confirm that is 100